.gitignore

```diff
@@ -33,6 +33,7 @@ frontend/test/ui/visual-regression/screenshots/screen
 .idea/
 .ijwb/
 *.iml
+.history/*
```

Comment:
If you can clean up your local repo without changing .gitignore then you do not need root OWNER approval :)

Reply:
You are right! I didn't realize that was part of a commit. I removed that line from .gitignore. So no more need for root OWNER  approval. Thx!